dockerfiles/Dockerfile.arm64v8

```diff
@@ -21,72 +9,103 @@ ENV FLB_VERSION 1.8.0
 
 ENV DEBIAN_FRONTEND noninteractive
 
-RUN apt update && \
-    apt install -y --no-install-recommends \
+RUN dpkg --add-architecture arm64
+RUN apt update && apt install -y --no-remove --no-install-recommends \
     build-essential \
     g++-aarch64-linux-gnu \
     gcc-aarch64-linux-gnu \
-    curl \
     ca-certificates \
     pkg-config \
     cmake \
     make \
-    tar \
     flex \
-    bison
+    bison \
+    dpkg-dev \
+    libssl-dev:arm64 \
+    libsasl2-dev:arm64 \
+    libsystemd-dev:arm64 \
+    libzstd-dev:arm64 \
+    zlib1g-dev:arm64 \
+    libpq-dev:arm64
 
-COPY --from=dep /usr/lib/aarch64-linux-gnu /usr/lib
-COPY --from=dep /lib/aarch64-linux-gnu /lib
-COPY --from=dep /usr/include /usr/include
+# postgresql-server-dev-11:arm64 has conflicts if installed using apt. hack here to manually force install with dpkg
+WORKDIR /tmp
+RUN apt download postgresql-server-dev-11:arm64
+RUN dpkg --force-all -i postgresql-server-dev-11_11.10-0+deb10u1_arm64.deb
```

Comment:
This will break, whenever the version changes. Switch it to postgresql-server*.deb or alike.

Reply:
Done.